tests/helpers/fd_utils.cc

```diff
@@ -78,8 +81,31 @@ namespace Pistache
 
         return std::distance(directory_iterator(fds_dir),
                              directory_iterator {});
+#else // fallback case, e.g. *BSD
+#ifndef OPEN_MAX
+#define OPEN_MAX	4096
+#endif
+        long maxfd;
+
+	maxfd = sysconf(_SC_OPEN_MAX);
+        if (maxfd < 0) // or if sysconf not defined at all
+            maxfd = getdtablesize();
+
+        if ((maxfd < 0) || (maxfd > 4*OPEN_MAX))
+            maxfd = OPEN_MAX;
+
+        int j, n = 0;
+        for(j = 0;  j < maxfd;  j++)
```

Comment:
It looks like this code hasn't been formatted with clang-format

Reply:
True... Ran clang-format -i